bin/webpack-configs.js

```diff
@@ -214,7 +214,7 @@ const getMainConfig = ( options = {} ) => {
 		],
 		resolve: {
 			...resolve,
-			extensions: [ '.js', '.ts', '.tsx' ],
+			extensions: [ '.js', '.jsx', '.ts', '.tsx' ],
```

Comment:
I don't really know why this is needed but I couldn't get `npm build` to work without it. :confused: